.github/ISSUE_TEMPLATE/Question.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Question
+about: The issue tracker is not for questions. Please use Stack Overflow or other resources for help.
```

Comment:
I thought we still are good to allow questions to be submitted here, however, not encouraged?

Reply:
Good point. Can you propose a better version of this text please?

Reply:
Feel free to edit the template directly, it may be the fastest way.